src/atoms/ComicCard.tsx

```diff
@@ -13,7 +14,7 @@ function ComicCard({ comic }: { comic: IComicData }) {
         />
       </CardBody>
       <CardHeader className="pb-0 pt-2 px-4 flex-col items-start">
-        <h4 className="font-bold text-large">{comic.name}</h4>
+        <h4 className="font-bold text-large">{comic.title}</h4>
```

Comment:
Why are you using `h4`?

Reply:
It was definitely meant to be h1😂 but i'll also deal with this in another branch